src/pages/Navgurukul/merakiAdmission/index.js

```diff
@@ -27,7 +27,7 @@ function Admission() {
       url: `${process.env.REACT_APP_MERAKI_URL}/users/me`,
       headers: {
         accept: "application/json",
-        Authorization: userToken !== "undefined" ? userToken : user.data.token,
+        Authorization: userToken !== undefined ? userToken : user.data.token,
```

Comment:
When the `Token` key is not present in the `localStorage`, `localStorage.getItem("Token");` will `return` `null` but `null !== undefined` is `true`.  So this is a case where you'll want to use `!=` instead because `null != undefined` will be `false`.  Alternatively, you can compare with `null` instead of `undefined`.  (See: https://developer.mozilla.org/en-US/docs/Web/API/Storage/getItem)
```suggestion
        Authorization: userToken != undefined ? userToken : user.data.token,
```